nixos/modules/config/i18n.nix

```diff
@@ -42,6 +42,16 @@
         '';
       };
 
+      extraLocales = lib.mkOption {
+        type = lib.types.listOf lib.types.str;
+        default = [ ];
+        example = [ "nl_NL.UTF-8" ];
+        description = ''
+          Additional locales that the system should support, besides the one
+          configured with {option}`i18n.defaultLocale`.
```

Comment:
I would mention `extraLocaleSettings` here too, to be complete.

Reply:
Alright, I've clarified the description :)